CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Code freeze date: YYYY-MM-DD
 
 ### Changed
 
+- Change default event name to string of event date when importing Hazard data from xarray [#795](https://github.com/CLIMADA-project/climada_python/pull/795)
```

Comment:
Could you please also add under `fixed` the issue that was fixed with this pull request please?

Reply:
I think just mentioning this under `Fixed` is enough